### DISCUSSION TITLE
Добавление способа удаления текущенго сообщения через inlinebutton

### DISCUSSION THREAD

**Discussion by Sterrist** [eternnoir/pyTelegramBotAPI#2291] [Ideas]
К примеру у нас есть сообщение к которому прикрепляется Inline кнопка, в некоторых случаях это кнопка 'Назад' тоесть по нажатии на нее сообщение с кнопкой удалиться, предлагаю добавить функцию удаление сообщения прямо в строке с кнопкой, Пример: backbtn = types.InlineKeyboardButton(text='Назад', delete_message=True) delete_message Если delete_message будет True то сообщение с этой кнопкой удалиться, если нет, то не удалиться

**Comment by coder2020official** [comment]
Useless. With this logic we can add many parameters. I'd follow what API tells us 

Furthermore, we have thought of adding methods to types, something like message.delete(); We came to conclusion that it would create a bigger mess
